pages/about.html

```diff
@@ -0,0 +1,140 @@
+                    <ul class="list">
+                        <li><a href="/pages/about.html">About us</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                        <li><a class="nav-btn">Code Campign</a></li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="mobile-menu" id="mobile-menu">
+                <nav>
+                    <ul id="nav-list">
+                        <li><a href="/">Home</a></li>
+                        <li><a href="/pages/about.html">About us</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                </nav>
+                <button id="close-modal" class="close-modal">
+                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+                        width="17" height="17" viewBox="0 0 17 17">
+                        <g></g>
+                        <path
+                            d="M9.207 8.5l6.646 6.646-0.707 0.707-6.646-6.646-6.646 6.646-0.707-0.707 6.646-6.646-6.647-6.646 0.707-0.707 6.647 6.646 6.646-6.646 0.707 0.707-6.646 6.646z"
+                            fill="#ffffff" />
+                    </svg>
+                </button>
+            </div>
+            <div class="about">
+                <div>
+                    <div class="mobile-nav" id="mobile-nav">
+                        <div class="menu">
+                            <div></div>
+                            <div></div>
+                            <div></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="headline container">
+                    <p class="title-sub">"Hello! Developers"</p>
+                    <h1 class="font-cocogoose title">CODE 4 EASY DEV BOOTCAMP 2022</h1>
+                    <p class="caption">
+                        Code 4 Easy Bootcamp takes all its students from knowing nothing about
+                        programming or web programming or anything related to independent
+                        software developer within 5 months of intensive learning and coding.
+                    </p>
+                    <p class="contact-info">Please contact us per Email for any further questions about Code 4 Easy
+                        Bootcamp</p>
+                    <a href="email:ishmaelkargbo5@outlook.com">Ishmaelkargbo5@gmail.com</a>
+                </div>
+            </div>
+        </header>
+        <section class="past logo-section">
+            <div class="container">
+                <div class="title-div">
+                    <p class="title">Code 4 Easy DevBootcamp Logo</p>
+                    <hr class="red-line">
```

Comment:
- [ ] OPTIONAL: Please center this text:
<img width="328" alt="Screenshot 2022-04-15 at 12 40 08" src="https://user-images.githubusercontent.com/77038610/163547901-92bf514d-a99b-4fab-88ad-6f85163e98db.png">

Reply:
OK Thanks 😊